packages/graph/hash_graph/lib/graph/src/store/postgres/knowledge/entity/read.rs

```diff
@@ -71,8 +71,48 @@ where
                             .resolve(tail_path_segments, context)
                             .await;
                     }
-                    "properties" => return resolve_properties(self.entity.properties(), tail_path_segments, context).await,
-                    "outgoingLinks" | "incomingLinks" => todo!("links are not supported yet, see https://app.asana.com/0/0/1202912966917503/f"),
+                    "properties" => {
+                        return resolve_properties(
+                            self.entity.properties(),
+                            tail_path_segments,
+                            context,
+                        )
+                        .await;
+                    }
+                    "outgoingLinks" => {
+                        // TODO: Use relation tables
+                        //   see https://app.asana.com/0/0/1202884883200942/f
```

Comment:
Just flagging that the task here is for "when querying types". We might want to update the title to reflect it would cover this as well

Reply:
Updated the title